data/campaigns/The_Rise_Of_Wesnoth/scenarios/09_Fallen_Lich_Point.cfg

```diff
@@ -344,7 +344,7 @@
         [/filter]
         [message]
             speaker=narrator
-            message= _ "This monolith was erected by me, ― (<i>chipped away</i>), first Mage of the good people of the Green Isle. By its power the Lich-Lord is bound in stone. To end the spell a noble of the line of Kings should utter the following..."
+            message= _ "This monolith was erected by me, ― (<i>chipped away</i>), first Mage of the good people of the Green Isle. By its power, the Lich-Lord is bound in stone. To end the spell, a noble of the line of Kings should utter the following..."
```

Comment:
I must have missed this in my recent review. I wonder if 'line of kings' should be lower-case.

Reply:
One could argue yes. There is a rule that says honorifics such as judge, king, etc, should be lower case if it does not follow a name. He is King Eldaric. A king. Meet Prince Haldric. A prince must fight. Want me to change king to lower case?

Reply:
If you could, yes.

Reply:
Wouldn't that also apply to "mage" in this line?

Reply:
I didn't even see that... It seems to be a title bestowed upon the inscriber, in which case I might argue for it to be 'First Mage'.

Reply:
Yeah, I'd agree that either both words should be capitlized or neither.